contracts/finance/andromeda-validator-staking/Cargo.toml

```diff
@@ -15,15 +15,17 @@ library = []
 testing = ["cw-multi-test"]
 
 [dependencies]
-cosmwasm-std = { workspace = true, features = ["staking"]  }
+cosmwasm-std = { workspace = true, features = ["staking", "stargate"]  }
```

Comment:
Is stargate still needed without the query?

Reply:
Oops. Forgot to remove that. It is not needed now